package.json

```diff
@@ -60,18 +60,17 @@
         "cli-table3": "^0.6.0",
         "conf": "^6.2.4",
         "googleapis": "^49.0.0",
-        "inquirer": "^7.3.3",
+        "inquirer": "^7.1.0",
         "inquirer-datepicker-prompt": "^0.4.2",
         "keytar": "^6.0.1",
-        "mongoose": "^5.10.15",
+        "mongoose": "^5.9.11",
         "mysql": "^2.18.1",
         "node-cron": "^2.0.3",
-        "node-mailer": "^0.1.1",
-        "nodemailer": "^6.4.16",
-        "omelette": "^0.4.15",
-        "ora": "^4.1.1",
+        "node-mailer": "^6.4.6",
```

Comment:
Kindly verify this package.
Seems like [this](https://www.npmjs.com/package/nodemailer) is the official package.

Reply:
ok sure

Reply:
done